### ISSUE TITLE
How to adjust the colormap in the plot_pole_colorbar function

### ISSUE THREAD

**Issue by hl04445567** [PmagPy/PmagPy#439]
I was wondering how can I adjust the color scheme (colormap) in plot_pole_colorbar function?

Sorry if this is a stupid question. I am new to Python and PmagPy.

**Comment by Swanson-Hysell** [comment]
Is your question how to change the range of values? Or how to change the colors that are used for the range of values?

There is currently no option to change the colormap within the plot_pole_colorbar of ipmag. At present it will use the default color map of matplotlib. However, it should be straightforward to put in such functionality so I will look to implement that functionality not.

**Comment by Swanson-Hysell** [comment]
There is now the option to change the color map. Here is an example where the colormap is changed to be 'plasma' (for a list of matplotlib colormaps look here: https://matplotlib.org/tutorials/colors/colormaps.html)

<img width="565" alt="screen shot 2018-09-26 at 12 41 19 pm" src="https://user-images.githubusercontent.com/4332322/46104942-91775f80-c189-11e8-9775-f0ac9dcd2cad.png">

Using this file:
[Laurentia_Mean_APWP.txt](https://github.com/PmagPy/PmagPy/files/2421301/Laurentia_Mean_APWP.txt)

And this code:
```
import pmagpy.pmag as pmag
import pmagpy.ipmag as ipmag
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
from mpl_toolkits.basemap import Basemap

Laurentia_APWP_T2012 = pd.read_csv('Laurentia_Mean_APWP.csv')

plt.figure(figsize=(8, 8))
pmap = Basemap(projection='ortho',lat_0=-25,lon_0=320,
               resolution='c',area_thresh=50000)
pmap.drawmapboundary(fill_color='white')
pmap.drawmeridians(np.arange(0,360,30))
pmap.drawparallels(np.arange(-90,90,30))

for n in range(0, len(Laurentia_APWP_T2012)):
     m = ipmag.plot_pole_colorbar(pmap, Laurentia_APWP_T2012['Plon_RM'][n],
                                  Laurentia_APWP_T2012['Plat_RM'][n],
                                  Laurentia_APWP_T2012['A95_RM'][n],
                                  Laurentia_APWP_T2012['Age'][n],
                                  0,
                                  540, colormap='plasma',
                                  markersize=80, color="k", alpha=1)
        
m = ipmag.plot_pole_colorbar(pmap, Laurentia_APWP_T2012['Plon_RM'][n],
                          Laurentia_APWP_T2012['Plat_RM'][n],
                          Laurentia_APWP_T2012['A95_RM'][n],
                          Laurentia_APWP_T2012['Age'][n],
                          0,
                          540, colormap='plasma',
                          markersize=80, color="k", alpha=1)

pmap.colorbar(m,location='bottom',pad="5%",label='Torsvik et al. (2012) running mean APWP age (Ma)')

plt.show()
```

**Comment by ltauxe** [comment]
that’s nice Nick - you could make color_map an argument as is done in pmagplotlib.plot_eq_cont
lisa

**Comment by Swanson-Hysell** [comment]
That is what the code does now following commit 0a53974eba8358ad8b8e2213ebf4886e6b630a15. There is an ```colormap``` argument with the default set to be the same default as matplotlib ('viridian').

```
def plot_pole_colorbar(mapname, plon, plat, A95, colorvalue, vmin, vmax, label='', colormap = 'viridis',
```

This function could be a bit more full-featured as there is a lot that needs to be wrapped around it as seen in the code example above. So I might look to make further improvements (however, perhaps that should be done in conjunction with a cartopy transition).

**Comment by hl04445567** [comment]
Wow! Thanks Nick and Lisa for the prompt response! 

I've got another stupid question. I tried to run your example code, but I got a TypeError:

plot_pole_colorbar() got an unexpected keyword argument 'colormap'

I think this is because the PmagPy package on my computer hasn't got the update you just made. I was wondering how should I update PamgPy?

I've ran 
  pip install --upgrade pmagpy --no-deps  
  pip install --upgrade pmagpy-cli --no-deps

Doesn't seems to work?
